### PR TITLE
Adding methods

### PR DESCRIPTION
**Pull request by joer9514**
Hey Bro! Added two methods

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,21 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """"A method to retrieve one object"""
```
[the remark this answers is not in the document]
Comment (joer9514) [reply models/engine/db_storage.py:79]: Following the clean code guide for python (pep8)

File: models/engine/db_storage.py [hunk 2]
```diff
@@ -74,3 +74,21 @@ def reload(self):
...
+       if cls not in self.__models_available:
+            return None
+        return self.__session.query(
+                self.__models_available[cls]).get(id)
+
+    def count(self, cls=None):
+        """A method to count the number of objects in storage"""
+      if cls is None:
+            total = 0
```
[the remark this answers is not in the document]
Comment (joer9514) [reply models/engine/db_storage.py:88]: Initializes to store the model

File: models/engine/file_storage.py [hunk 3]
```diff
@@ -68,3 +68,21 @@ def delete(self, obj=None):
...
+                return all_objs[k]
+        return None
+
+    def count(self, cls=None):
+        """A method to count the number of objects in storage"""
+        if cls is None:
+            return len(self.__objects)
+        if cls in self.__models_available:
+            return len(self.all(cls))
+        return -1
```
[the remark this answers is not in the document]
Comment (joer9514) [reply models/engine/file_storage.py:88]: would not affect

**Comment by FabianMosquera** [comment]
I see no drawbacks

**Pull request merged**